### ISSUE TITLE
Warnings shown when running gorpipe

### ISSUE THREAD

**Issue by gumson** [gorpipe/gor#1]
There are several warning which popup when running gorpipe related to reflection. Gorpipe built using instructions from repo but still getting these errors.

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/Users/gummi/source/gor/gortools/build/install/gor-scripts/lib/guice-4.2.2.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Chrom	gene_start	gene_end	Gene_Symbol
chr1	131024	134836	RP11-34P13.10
```

The operation finishes correctly, yet still an eyesore.

**Comment by Shicheng-Guo** [comment]
Hi Gumson, What's the final solution or comment to this satiation?

**Comment by snorristurluson** [comment]
This warning needs to be addressed in Guice, and it has taken longer for a fix to this than we expected (see https://github.com/google/guice/issues/1133).

A workaround is add this environment variable before running:
`export GOR_GORPIPE_OPTS="--add-opens java.base/java.lang=ALL-UNNAMED"`

We'll look into our options, but we have not seen any adverse effects of this and hopefully we can upgrade Guice soon.

**Comment by gmagnu** [comment]
This has been fixed in GOR 2.6.
